KustoQueryBuilder/Clauses/ProjectClause.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace KustoQueryBuilder.Clauses;
+
+public class ProjectClause : AbstractClause
+{
+    public IEnumerable<string> Columns { get; }
+
+    public ProjectClause(IEnumerable<string> columns)
+    {
+        if (!columns.Any())
```

Comment:
columns can be null here

Reply:
We have enabled Nullable reference types in the project so by default all reference types must have a value.